source/braille.py

```diff
@@ -878,10 +886,15 @@ def _addFieldText(self, text, contentPos):
 		self.rawTextTypeforms.extend((louis.plain_text,) * textLen)
 		self._rawToContentPos.extend((contentPos,) * textLen)
 
-	def _addTextWithFields(self, info, formatConfig, isSelection=False):

```

Comment:
Another doc string missing which might be useful somehow, but I'm overly nagging now.